src/db/userStore.js

```diff
@@ -4,12 +4,12 @@ import knex from './knex.js';
 
 export const RFID_SALT = '$2b$15$yvDy89XRQiv1e4M6Vn2m5e';
 
-const rowToUser = (row) => {
+export const rowToUser = (row) => {
 	if (row !== undefined) {
 		return {
 			userId: row.userid,
 			username: row.name,
-			fullName: row.realname,
+			fullName: row.realname || row.name,
```

Comment:
`realname` is more often NULL (1 638 rows) than a value (1 371 rows) in production data

Reply:
Real API fix should be done. Issue #120 tracking

Reply:
Did the null break something here? Just make sure to remove this || now or later.